### DISCUSSION TITLE
Missing entitlements on iOS (Apple sign in, aps-environment)

### DISCUSSION THREAD

**Discussion by miladdev85** [codemagic-ci-cd/codemagic-docs#1328] [Q&A] [answered]
We just noticed that when we build through Codemagic, the app has some missing entitlements when checking the metadata in TestFlight:

- com.apple.developer.applesignin
- com.apple.developer.associated-domains
- keychain-access-groups
- aps-environment

When built locally, all entitlements are there. Any suggestions?

Build id: `63c7cdcccae9f090ee4365a6`

**Comment by himesh-cm** [comment, marked as answer]
Hi @miladdev85, Can you please try Xcode 13.4.1 with the below build command?
```
xcode-project build-ipa \
  --workspace "$CM_BUILD_DIR/ios/$XCODE_WORKSPACE" \
  --scheme "$XCODE_SCHEME" \
```

**Reply by miladdev85** [reply]
@himesh-cm That was it, you rock! 👑
Thank you so much!
